src/pages/login.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import Layout from '../layout/Layout';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import { OPERATOR_COOKIE } from '../constants';
+import { ErrorInfo } from '../types';
+import { deleteCookieOnServerSide } from '../utils/index';
+
+const title = 'Login - Fares data build tool';
+const description = 'Login page of the Fares data build tool';
+
+interface LoginProps {
+    errors: ErrorInfo[];
+}
+
+const Login = ({ errors = [] }: LoginProps): ReactElement => {
+    return (
+        <Layout title={title} description={description}>
```

Comment:
So BaseLayout would go here and we can pass `errors` to it as well

Reply:
Now using BaseLayout.